catkin_tools/resultspace.py

```diff
@@ -59,6 +59,7 @@ def get_resultspace_environment(result_space_path, quiet=False, cached=True):
     # Use fallback shell if using a non-standard shell
     if shell_name not in ['bash', 'zsh']:
         shell_name = 'bash'
+        shell_path = "/bin/bash"
```

Comment:
This is what the other PR did initially with https://github.com/pjreed/catkin_tools/commit/4e730b33ccaeb912f023dbb6e536b4851dbc18bf, but there were some concerns about `sh` being maybe a better default. As I said I would stick with `bash` until it is clear how `sh` should behave.